Birthday-2024-Project/Scripts/GameManager.gd

```diff
@@ -203,10 +205,16 @@ func _do_place_held_piece():
 			held_piece.return_piece()
 		held_piece = null
 		return # Piece does not fit
+
+	#Prevents rotation while the timer is running
+	placing_piece = true
+	await get_tree().create_timer(.4).timeout	
```

Comment:
just a comment: one of my pet peeves is magic numbers. Could 0.4 be made into a constant value?

Reply:
Should be all good now, added a variable for how long the delay is

Reply:
you are doubling the code now, remove lines 209, 210, 211, 212